### PR TITLE
just a start...

### PR DESCRIPTION
**Pull request by soyjavi**
- use nodejs lts (v10) and remove some vulnerabilities
- use linter and airbnb best practices
- refactor in `invoicer.js`
- new module `fetch` (retrieving a json file... and control the async/await inside it)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,25 +4,17 @@
...
+    "format": "prettier --write 'packages/**/*.js'"
   },
   "repository": "https://github.com/lncm/donations",
   "author": "meeDamian",
   "license": "MIT",
   "private": false,
   "engines": {
-    "node": ">=11",
-    "npm": ">=6"
+    "node": ">=10"
```
Comment (meeDamian) [inline package.json]: Shouldn't that be sth like `=10`?  Doesn't it go for the newer major, if available?

**Review by meeDamian — APPROVED** [review]

**Comment by meeDamian** [comment]
6k LOC removed 😂

File: package.json (hunk 1 above)
Comment (soyjavi) [reply package.json]: Will use `v10` + `minor` + `patch`

**Comment by AnotherDroog** [comment]
Wow @soyjavi !

**Comment by AnotherDroog** [comment]
>6k LOC removed 😂

lock file 🤣

**Pull request merged**